doc/Logging.md

```diff
@@ -0,0 +1,16 @@
+How does Happy test log work
+============================
+
+## Happy produced logs
+  * Happy host logs
+  * Happy device logs
+
+## Where to store Happy logs
+
+Both Happy host and device logs location are processed in happy/Driver.py as below:
+  * if HAPPY_LOG_DIR is found in os environment variable, happy uses HAPPY_LOG_DIR's value as log folder location to store happy test logs. For example, openweave-core tests store happy test logs along with feature folder under build folder.
+  * if HAPPY_LOG_DIR is not found in os environment variable, happy uses "default_happy_log_dir" from happy/conf/main_config.json as default log folder location to store happy test logs
```

Comment:
- `default_happy_log_dir` in code font
- Link to happy/conf/main_config.json in the repo
- ...as **the** default log folder..."

Reply:
done